src/style.css

```diff
@@ -1 +1,21 @@
 /* styles go here */
+
+.block {
+  margin-bottom: 20px;
+}
+
+.fieldset {
+  padding-top: 10px;
+}
+
+button {
```

Comment:
let's create class for it, but as for me.. it needless, and u can remove this style)

Reply:
I'm sorry, what exactly do you mean, because i didn`t quite understand you(

Reply:
Why do you need to style this button?)

Reply:
oh okey, i realized my mistake